content/applications/hr/employees/new_employee.rst

```diff
@@ -34,45 +46,46 @@ Required fields
+     Position`, if desired.
 
-     An example where this may be applicable is if someone is hired for a sales representative
-     position configured as :guilabel:`Sales Representative` in the *Recruitment* app, and that is
-     selected for the drop-down :guilabel:`Job Position` field.
+     For instance, if someone is hired for a sales representative position configured as
+     :guilabel:`Sales Representative` in the *Recruitment* app, that can be selected in the
+     drop-down :guilabel:`Job Position` field.
 
-     In the typed in job position field beneath the employee's name, the position could be more
-     specific, such as `Sales Representative - Subscriptions` if the employee is focused solely on
-     subscription sales.
+     In the typed-in :guilabel:`Job Position` field beneath the :guilabel:`Employee's Name` field,
+     the position could be more specific, such as `Sales Representative - Subscriptions` if the
+     employee is focused solely on subscription sales.
 
      .. image:: new_employee/job-description-fields.png
         :align: center
         :alt: Both job position fields entered but with different information.
 
-- Tags: Click on a tag in the drop-down menu to add any tags applicable to the employee. Any tag can
-  be created in this field by typing it in. Once created, the new tag is available for all employee
-  records. There is no limit to the amount of tags that can be added.
-- Work Contact Information: Enter the employee's :guilabel:`Work Mobile`, :guilabel:`Work Phone`,
-  :guilabel:`Work Email`, and/or :guilabel:`Company` name.
-- :guilabel:`Department`: Select the employee's department from the drop-down menu.
-- :guilabel:`Job Position`: Select the employee's job position from the drop-down menu. Once a
-  selection is made, the :guilabel:`Job Position` field beneath the employee's name automatically
-  updates to reflect the currently selected job position. These positions are from the
-  :doc:`Recruitment <../../hr/recruitment/new_job/>` application, and reflect the currently \
-  configured job positions.
-- :guilabel:`Manager`: Select the employee's manager from the drop-down menu.
-- :guilabel:`Coach`: Select the employee's coach from the drop-down menu.
-- :guilabel:`Next Appraisal Date`: This date automatically populates with a date that is computed
-  according to the settings configured in the *Appraisals* application. This date can be modified
-  using the calendar selector.
+- :guilabel:`Tags`: select a tag from the drop-down menu to add relevant tags to the employee. Any
+  tag can be created in this field by typing it in. Once created, the new tag is available for all
+  employee records. There is no limit to the amount of tags that can be added.
+- Work Contact Information: enter the employee's :guilabel:`Work Mobile`, :guilabel:`Work Phone`,
+  :guilabel:`Work Email`, and/or :guilabel:`Company` name, if not already auto-populated.
```

Comment:
I totally agree with this! I just never made it a GUI because technically, it does not get one. I HAPPILY am changing this since I agree, it looks better. I just didn't think I was "allowed" to since it is technically incorrect.